compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -2165,7 +2165,8 @@ const char *OMR::SymbolReferenceTable::_commonNonHelperSymbolNames[] =
    "<atomicCompareAndSwapReturnStatus>",
    "<atomicCompareAndSwapReturnValue>",
    "<jProfileValueSymbol>",
-   "<jProfileValueWithNullCHKSymbol>"
+   "<jProfileValueWithNullCHKSymbol>",
+   "<defaultValue>"
```

Comment:
There are actually a few strings missing between `jProfileValueWithNullCHKSymbol` and `defaultValue`.  Could you add them as well please?
```
"<j9VMThreadTempSlotFieldSymbol>",
"<computedStaticCallSymbol>",
"<j9VMThreadFloatTemp1Symbol>",
"<J9JNIMethodIDvTableIndexFieldSymbol>",
"<defaultValueSymbol>"
```

This table isn't enabled yet.  The non-helper enum was in the process of being split for downstream projects but the final pieces weren't finished up.  This is a good reminder (to me) to do that.